servlet-feature-pack/src/main/resources/content/domain/configuration/default-server-logging.properties

```diff
@@ -0,0 +1,44 @@
+#
```

Comment:
Is there a reason for this being added? It should be coming from core I would think.

Reply:
@jamezp Without this file, we would be generating different logging properties for the domain configuration in full build/dist and servlet-build/servlet-dist. I included it here to match both distributions.

Reply:
Oh that's right. Sorry, I forgot that in full we remove the console-handler for domain servers.